code/modules/reagents/recipes.dm

```diff
@@ -49,6 +49,7 @@
 	var/result = null
 	var/list/required_reagents = list()
 	var/list/catalysts = list()
+	var/list/alchemy_catalysts = list()
```

Comment:
Is this var used for anything?

Reply:
Nope that idea was scrapped after a few days of tweaking to no avail on it working